### ISSUE TITLE
The csv file name problem and the problems encountered in the new paper

### ISSUE THREAD

**Issue by Dongj909** [ChenyangLEI/flash-reflection-removal#12]
Hello, Thank you very much for providing this excellent work. I replicated your experiment under flash-reflection-removal / tree / main / torchfrr. However, it is doubtful about the names of some files. I would like to ask the meaning of each name in your row coordinates, grateful.
<img width="874" alt="894f3cff7f6be8d72a70e7b19ce7150" src="https://github.com/ChenyangLEI/flash-reflection-removal/assets/128920615/86ddcbfa-8cb6-41c3-b90b-19d2e138fa3c">
I want to know which data is the corresponding data published in the paper.
![e1411b434eb4da7bd604c5e4a449b84](https://github.com/ChenyangLEI/flash-reflection-removal/assets/128920615/e1a19fac-4b89-4877-a5df-6621ad3433b6)
There is the test does not know what the problem, flow results are all 0, in the training results are not so.This leads to the results of perc _ rgb _ t and perc _ rgb _ r are far from the training results.
<img width="143" alt="19f47edfcb22776c4dba258ff7d3e37" src="https://github.com/ChenyangLEI/flash-reflection-removal/assets/128920615/8a58467c-5789-4ad1-831a-2bc1fb5498db">
Looking forward to your reply, thanks again

**Comment by xjiangan** [comment]
Sorry for the confusion.

The results for the aligned images come from previous TensorFlow code. Experiments in torchfrr primarily contribute to the following paper, which focuses on handheld photography with misaligned flash and non-flash pairs:
```
@ARTICLE{10250963,
  author={Lei, Chenyang and Jiang, Xudong and Chen, Qifeng},
  journal={IEEE Transactions on Pattern Analysis and Machine Intelligence}, 
  title={Robust Reflection Removal With Flash-Only Cues in the Wild}, 
  year={2023},
  volume={45},
  number={12},
  pages={15530-15545},
  doi={10.1109/TPAMI.2023.3314972}}
```

Regarding the result format:
Columns:
 - The prefix "perc" refers to perceptual loss, and "PSNR", "SSIM", "LPIPS" are the evaluation metrics.
 - The suffix "t" indicates transmission prediction, "r" indicates reflection prediction, "t0" indicates directly using input ambient as transmission. 

Rows:
 - Prefix "ma" denotes the original misaligned inputs, used for most evaluations.
 - Prefix "masa" denotes misaligned but preprocessed with SIFT homography alignment, used in ablation studies. 

For example, our complete model "dwpwc" utilizes depth-based misaligned training data and an optical flow alignment module. The results are taken from the average of rows madeskalign_0206 and madeskalign_0218. We use the columns psnr_rgb_t, ssim_rgb_t, and lpips_rgb_t. Furthermore, to ensure the robustness and reproducibility of our results, we perform multiple trials for each experiment and report the mean values.
